src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
```

Comment:
IMO the overhead of this method is not too important as we really don't expect it to be used in production. I would be down for updating the javadoc to reflect the fact that you call this method in performance-critical code at your own risk.